analysis_options.yaml

```diff
@@ -1,12 +1,23 @@
+analyzer:
+  exclude:
+    - mobx_examples/lib/**/*.g.dart
+    - mobx_codegen/test/data/**.dart
+  # TODO: migrate mobx to enable these
+  # strong-mode:
+  #   implicit-casts: false
```

Comment:
What is preventing from using this right now ?

Reply:
Enabling both triggers more than 100 errors

They should be fixed, but probably not as part of this PR